### ISSUE TITLE
Temporary Whitelist

### ISSUE THREAD

**Issue by madhavarshney** [tbroadley/github-spellcheck-cli#81]
Another useful option for a misspelled word is temporary whitelist, for the current repository.  This is useful in projects like webpack.js.org where you wouldn't necessarily want to permanently whitelist the GitHub usernames in the contributors section of each page but only whitelist them for that repo.

Thanks, Madhav.

**Comment by tbroadley** [comment]
Hey! The tool used to have a command to whitelist a word only for the current run. I got rid of it because I found I was using it too much instead of permanently whitelisting words, which was overall reducing the benefit of having the persistent whitelist. My current thinking for things like names is that, if it's spelled correctly in the repo I'm currently reviewing, it probably means I'd accept the spelling in a different repo too :slightly_smiling_face:
